# Семинар 14. Всё нормально!

дата: 12 декабря 2016

Конспект: Фадеева Екатерина

## Задача 1. 

$X\in [0;1]$

$Y=-lnX \in [0; +\infty)$

*Найти функцию плотности:*

$f(y)={\large \frac{dF(y)}{dy}=\frac{dP(Y\leqslant y)}{dy}=\frac{dP(-lnX\leqslant y)}{dy}=\frac{dP(e^{-lnX}\leqslant e^y)}{dy}=\frac{dP(X^{-1}\leqslant e^y)}{dy}=\frac{dP(X\geqslant e^{-y})}{dy}=\frac{d(1-e^{-y})}{dy}}=-e^-{}^y(-1)=e^-{}^y$ 

$f(y) =
\begin{cases}
0, & \text{если $y<0$;} \\
e^-{}^y, & \text{если $y\geqslant 0$.}
\end{cases}$     (на седьмом переходе в решении выше мы можем заметить это условие)

## Ц.П.Т. + Дельта-метод = ВСЁ НОРМАЛЬНО!

### Решим задачу:

$X_1, X_2, \dots, X_n$ независимы

X |  -2 | 4
-|-|-
P | 0.5 | 0.5

а) как примерно распределено $\bar X_{100}$?

б) как примерно распределено ${\large \frac{1}{\bar X^2+1}}$?

$E(\bar X_{100})=E(X_i)=1$

$Var(X_{100})=4\cdot0.5+16\cdot0.5-1=10-1=9$

$Var(\bar X_{100})={\large \frac {9}{100}}$

a) $\bar X_{100} \sim N(1; 0.09)$

б) Разложим в ряд Тейлора в окрестности $E(\bar X)$:

${\large \frac{1}{\bar X^2+1}=f(\bar X)=\frac {1}{2}-\frac {1}{2}(\bar X -1) + \displaystyle o(\bar X)}$

$E(f(\bar X))\approx {\large \frac {1}{2}}$


$Var(f(\bar X)) \approx \frac {1}{4} \cdot Var(\bar X - 1)= \frac {1}{4} \cdot\frac {9}{100}$

${\large \frac{1}{\bar X^2+1} \sim N(\frac {1}{2}; \frac {9}{400})}$

*Чем больше количество, тем меньше дисперсия ($\href{http://sernam.ru/book_tp.php?id=65}{см. неравенство Чебышёва}$)* 

То, что мы только что использовали, и называется  **ДЕЛЬТА-МЕТОД**!

Если последовательность ${\LARGE \frac {z_n - \mu}{\sqrt {\frac {\sigma^2}{n}}}}{\large \xrightarrow {dist} N(0;1)
}$, то    ${\LARGE \frac {f(z_n) - f(\mu)}{\sqrt {\frac {f'(\mu)^2\sigma^2}{n}}}}{\large \xrightarrow {dist} N(0;1)
}$

Другая формулировка:

Если ${\large z_n \sim N(\mu;\frac {\sigma^2}{n})}$,

то ${\large f(z_n) \sim N(f(\mu); (f'(\mu))^2\frac {\sigma^2}{n})}$

## Задача 2. 

$X_1,X_2, X_3...$ независимы

X |  2 | 0.5
-|-|-
P | 0.5 | 0.5

$R=\sqrt[100]{X_1\cdot X_2\cdot X_3 ... \cdot X_{100}}$

$P(R\in [0.9;1.1])=?$ (какова вероятность того, что эта случайная величина лежит от 0,9 до 1,1?)

Важно понять, как распределена $R$.

$Y=lnR=\frac{1}{100}(lnX_1+lnX_1+lnX_3+...+lnX_{100})$

По Ц.П.Т. распределено нормально. 

$L_i=lnX_i$ |  ln2 | -ln2
-|-|-
P | 0.5 | 0.5

$E(L_i)=\frac {1}{2}ln2-\frac {1}{2}ln2=0$

$Var(L_i)=E({L_i}^2)-(E(L_i))^2=\frac {1}{2}(ln2)^2+\frac{1}{2}(ln2)^2=(ln2)^2$

$Y={\large \frac {L_1+L_2+L_3+...+L_{100}}{100}}$

$E(Y)=0$

$Var(Y)=\frac {1}{100^2}(Var(L_1+L_2+...+L_100))=\frac {1}{100^2} \cdot 100(ln2)^2={\large \frac {(ln2)^2}{100}}$

$Y\sim N(0; {\large \frac {(ln2)^2}{100}})$

$R=e^y=1+Y+\displaystyle o(Y) \approx 1+Y$

$E(R) \approx 1$

$Var(R) \approx {\large \frac {(ln2)^2}{100}}$

$R \sim N(1; {\large \frac {(ln2)^2}{100}})$

In [23]:
import math

In [30]:
(math.pow(math.log(2), 2))/100

0.004804530139182014

$R\sim N(1; 0.005)$

$P(0.9\leqslant R\leqslant 1.1)=P({\large \frac {-0.1}{\sqrt {0.005}}\leqslant \frac {R-1}{\sqrt {0.005}}\leqslant \frac {0.1}{\sqrt {0.005}}})$

In [31]:
0.1/math.sqrt(0.005)

1.4142135623730951

$P(-1.41\leqslant {\large \frac {R-1}{\sqrt {0.005}}}\leqslant 1.41)$ для $N(0;1)$

$P(-1.41\leqslant {\large \frac {R-1}{\sqrt {0.005}}}\leqslant 1.41)=0.9207 \cdot 2 -1=0.84$

In [43]:
0.9207*2-1

0.8413999999999999

Как отмасштабировать: ${\LARGE \frac {\bigstar - E(\bigstar)}{\sqrt {Var(\bigstar)}}}$ *(здесь должна была быть черепашка)*